config/manifests/bases/awx-resource-operator.clusterserviceversion.yaml

```diff
@@ -61,6 +62,11 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:text
         - urn:alm:descriptor:com.tectonic.ui:advanced
+      - displayName: Request Timeout
+        description: Time in seconds to wait for a response from the AWX API
+        path: request_timeout
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
```

Comment:
maybe add `urn:alm:descriptor:com.tectonic.ui:advanced` too ?
What do you think ?

Reply:
agreed